05/shkulkova/src/components/Header/HotelParams/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
+
+export const HotelParams = () => {
+  const [opened, setOpened] = useState(false);
+  const [adults, setAdults] = useState(0);
+  const [children, setChildren] = useState(0);
+  const [rooms, setRooms] = useState(1);
+
+  return (
+    <>
+      <button className="params_button" onClick={() => setOpened(!opened)}>
+        {adults} Adults — {children} Children — {rooms} Room
+      </button>
+      {opened && (
+        <div className="params_container">
+          <div className="params_item">
+            <p className="params_item_title">Adults</p>
+            <div className="item_counter">
+              <button
+                className="params_item_btn"
+                onClick={() => setAdults(adults - 1)}
```

Comment:
could you create separate functions and not to write directly in the render part of the component?